### DISCUSSION TITLE
Quarto 1.4.549: Section link appearing at top of page (book): how to remove it?

### DISCUSSION THREAD

**Discussion by pglpm** [quarto-dev/quarto-cli#8453] [Q&A] [answered]
### Description

I've updated quarto to 1.4.549 and recompiled a book-style website. Looking with **quarto-preview** I notice that every page/chapter now has a small header reporting the "part" from the `_quarto.yml` (already visible on the left bar) and the title of the chapter itself. See snippet below, where the new header has been surrounded by a red ellipse.

- Is this something that only appears in quarto-preview?
- If not, how to eliminate this header? 

Thank you!

![Screenshot_20240126_123747](https://github.com/quarto-dev/quarto-cli/assets/9976691/efb10f93-1827-4d0e-9e92-546798decb4c)

**Comment by mcanouil** [comment, marked as answer]
That's breadcrumbs, see <https://quarto.org/docs/websites/website-navigation.html#breadcrumbs>.

**Reply by pglpm** [reply]
Fantastic, thank you! Didn't know the name and wasn't managing to find this in the manual :)
